lib/ui/text.dart

```diff
@@ -90,7 +90,10 @@ enum TextAlign {
   right,
 
   /// Align the text in the center of the container
-  center
+  center,
+
+  /// Stretch each line of text to fill the width of the container
+  justify
```

Comment:
Yes. Also verified on device that "justify" has the desired effect: 

![screenshot_20170119-150424](https://cloud.githubusercontent.com/assets/1227763/22129106/f454cdce-de58-11e6-939f-72965774f550.png)